src/extension.ts

```diff
@@ -108,29 +105,30 @@ function logToConsole(consoleMessage: string, fromDebugger: boolean = false) {
     if (fromDebugger) {
         // STDERR messages of the `error` variant. These deserve to be shown as an error message in the UI also.
         // This filter might capture more than expected, but since RUST_LOG messages can take many formats, it seems that this is the safest/most inclusive.
-        if (consoleMessage.includes(ConsoleLogSources.error)) {
+        if (consoleMessage.startsWith(ConsoleLogSources.error)) {
+            vscode.debug.activeDebugConsole.appendLine(consoleMessage);
             vscode.window.showErrorMessage(consoleMessage);
         } else {
             // Any other messages that come directly from the debugger, are assumed to be relevant and should be logged to the console.
             vscode.debug.activeDebugConsole.appendLine(consoleMessage);
         }
-    } else if (consoleMessage.includes(ConsoleLogSources.console.toLowerCase())) {
+    } else if (consoleMessage.startsWith(ConsoleLogSources.console.toLowerCase())) {
         vscode.debug.activeDebugConsole.appendLine(consoleMessage);
     } else {
         switch (consoleLogLevel) {
             case ConsoleLogSources.debug: //  Log Info, Error AND Debug
                 if (
-                    consoleMessage.includes(ConsoleLogSources.console.toLowerCase()) ||
-                    consoleMessage.includes(ConsoleLogSources.error) ||
-                    consoleMessage.includes(ConsoleLogSources.debug)
+                    consoleMessage.startsWith(ConsoleLogSources.console.toLowerCase()) ||
```

Comment:
`probe-rs-debug` is already lower case, why is there a `.toLowerCase()` transform on it in multiple (at least 3) places?

Reply:
A hang over from when that string was uppercase. Good spotting ... I will fix it.